app/src/androidTest/java/chat/rocket/android/Config.kt

```diff
@@ -0,0 +1,23 @@
+package chat.rocket.android
+
+class Config {
+    companion object {
+        const val communityServerUrl: String = "https://open.rocket.chat"
+        const val currentServer: String = "https://open.rocket.chat"
+        const val validServer: String = "https://open.rocket.chat"
+        const val SERVER: String = "open.rocket.chat"
+        const val serverUrl: String = "serverUrl"
+        const val NAME: String = "user"
+        const val EMAIL: String = "abc@gmail.com"
+        const val USERNAME: String = "user121"
+        const val PASSWORD: String = "123456"
+        const val CODE = "1234"
+        const val USER_ID = "user_id"
+        const val AUTH_TOKEN = "auth_token"
+        const val TERMS_OF_SERVICE: String = "Terms of Service"
+        const val PRIVACY_POLICY: String = "Privacy Policy"
+        const val termsOfServiceUrl: String = "https://open.rocket.chat/terms-of-service"
+        const val privacyPolicyUrl: String = "https://open.rocket.chat/privacy-policy"
```

Comment:
Very small thing........  for these two url's, it probably makes sense to concatenate with one of the previous variables declared above?  ```currentServer + "/terms-of-service"```, for example...

Reply:
Sure. Will update👍